test/functional/wallet_multiwallet.py

```diff
@@ -147,7 +143,7 @@ def wallet_file(name):
 
         self.restart_node(0, extra_args)
 
-        assert_equal(set(map(lambda w: w['name'], self.nodes[0].listwalletdir()['wallets'])), set(['', 'w3', 'w2', 'sub/w5', 'w7', 'w7', 'w8_copy', 'w1', 'w8', 'w']))
+        assert_equal(set(map(lambda w: w['name'], self.nodes[0].listwalletdir()['wallets'])), set(['', 'w3', 'w2', os.path.join('sub', 'w5'), 'w7', 'w7', 'w8_copy', 'w1', 'w8', 'w']))
```

Comment:
In windows it is possible to set `-wallet=sub/w5, `listwallets` gives `sub/w5` but `listwalletdir` gives `sub\w5`?

Reply:
Yes, unless we change to calling `fs::path::generic_string()` instead of calling `fs::path::string()`